install/ci-vm/ci-linux/ci/bootstrap

```diff
@@ -20,6 +20,9 @@ if [ "${maintenance}" == "True" ]; then
     exit;
 fi
 
-wget "${runCIURL}"
+# Copy runCI to local folder
+cp ${runCIFile}/* ${runCIFile}
```

Comment:
Copy to the same spot? That'll work.

It needs to be copied from one place (the NFS share) to the directory we're currently in.

Besides, the current command isn't correct. We don't need a copy of all files, that variable should point to a single file.